backend/cmd/server/repository/resources/graphs/auth_flow_config_basic_google_github_sms.json

```diff
@@ -127,6 +127,11 @@
                     "name": "lastName",
                     "type": "string",
                     "required": false
+                },
+                {
```

Comment:
This is required. Otherwise mobile will not be saved in user profile in MFA flows